extensions/amp-render/1.0/amp-render.js

```diff
@@ -212,7 +214,51 @@ export class AmpRender extends BaseElement {
       },
       'onReady': () => {
         this.toggleLoading(false);
-        this.togglePlaceholder(false);
+        if (this.element.getAttribute('layout') !== Layout.CONTAINER) {
+          this.togglePlaceholder(false);
+          return;
+        }
+        if (!this.getPlaceholder()) {
+          user().error(TAG, 'placeholder required with layout="container"');
+          return;
+        }
+
+        let height;
+        this.measureMutateElement(
+          () => {
+            height = computedStyle(
+              this.getAmpDoc().win,
+              this.element
+            ).getPropertyValue('height');
+          },
+          () => {
+            setStyles(this.element, {
+              'overflow': 'hidden',
+              'height': height,
+            });
+          }
+        ).then(() => {
+          let containerHeight;
+          this.measureMutateElement(
```

Comment:
This can be called inside `measureElement` because the given mutate block (`this.attemptChangeHeight`) already schedules the resize request on its own.